packages/freeimage/build.sh

```diff
@@ -0,0 +1,24 @@
+TERMUX_PKG_HOMEPAGE=https://freeimage.sourceforge.io
+TERMUX_PKG_DESCRIPTION="FreeImage is a library project for developers who would like to support popular graphics image formats."
```

Comment:
If you shorten `TERMUX_PKG_DESCRIPTION` to for example `Library project for developers who would like to support popular graphics image formats` the build will continue a bit further.

Reply:
I updated it.